tests/test_orders.py

```diff
@@ -446,8 +460,10 @@ def test_update_order__failure_not_own_order(self):
             "order": {
                 "order_id": order_id,
                 'items': [
-                    {'name': 'mario', 'price': 20.0, 'quantity': 5},
-                    {'name': 'GINO', 'price': 30.20, 'quantity': 1}
+                    {'item_id': '429994bf-784e-47cc-a823-e0c394b823e8',
+                     'price': 20.0, 'quantity': 5},
+                    {'item_id': '577ad826-a79d-41e9-a5b2-7955bcf03499',
+                     'price': 30.20, 'quantity': 1}
                 ],
                 'delivery_address': addr_B.json()["address_id"],
                 'user': '86ba7e70-b3c0-4c9c-8d26-a14f49360e47'
```

Comment:
Poiché con la PR si va a risolvere un bug, è necessario scrivere uno o più test che dimostrino che il vecchio codice era buggato e il nuovo no. In questo caso quindi sarà necessario scrivere un test per la POST e uno per la PATCH che vada ad inviare degli items che non esistano e che riceva un 400.